packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -162,15 +162,15 @@ export class PreferencesController extends BaseController<
     const { identities, lostIdentities } = this.state;
     const newlyLost: { [address: string]: ContactEntry } = {};
 
-    for (const identity in identities) {
+    for (const identity of Object.keys(identities)) {
       if (!addresses.includes(identity)) {
         newlyLost[identity] = identities[identity];
         delete identities[identity];
       }
     }
 
     if (Object.keys(newlyLost).length > 0) {
-      for (const key in newlyLost) {
+      for (const key of Object.keys(newlyLost)) {
         lostIdentities[key] = newlyLost[key];
       }
     }
```

Comment:
Can we iterate over `Object.entries` here? Also we don't need to check for `Object.keys(...).length` first.

``` suggestion
    for (const [key, value] of Object.entries(newlyLost)) {
      lostIdentities[key] = value;
    }
```

Reply:
Oh really? Weird. Yeah, since it's less than a percentage point, I think it's okay to adjust the threshold value.